server/server.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable no-console */
 /* eslint-disable camelcase */
+/* istanbul ignore file */
```

Comment:
why are we ignoring server.js? I know it can't really be tested but it feels like it should be part of the coverage